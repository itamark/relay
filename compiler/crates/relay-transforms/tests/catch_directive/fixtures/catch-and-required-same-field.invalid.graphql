# expected-to-throw
fragment EnabledFoo on User {
  name @catch @required(action: THROW)
}
