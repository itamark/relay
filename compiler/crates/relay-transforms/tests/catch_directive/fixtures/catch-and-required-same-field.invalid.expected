==================================== INPUT ====================================
# expected-to-throw
fragment EnabledFoo on User {
  name @catch @required(action: THROW)
}
==================================== ERROR ====================================
✖︎ @catch is not supported with required directive on same field.

  catch-and-required-same-field.invalid.graphql:3:8
    2 │ fragment EnabledFoo on User {
    3 │   name @catch @required(action: THROW)
      │        ^^^^^^
    4 │ }
